sbaid/model/database/global_sqlite.py

```
@@ -33,75 +37,78 @@ class GlobalSQLite(GlobalDatabase):
-        await self._connection.executescript("""PRAGMA foreign_keys = ON;
-DROP TABLE IF EXISTS project;
-DROP TABLE IF EXISTS result;
-DROP TABLE IF EXISTS result_tag;
-DROP TABLE IF EXISTS tag;
-DROP TABLE IF EXISTS snapshot;
-DROP TABLE IF EXISTS cross_section_snapshot;
-DROP TABLE IF EXISTS lane_snapshot;
-DROP TABLE IF EXISTS vehicle_snapshot;
-
-CREATE TABLE project (
-    id TEXT PRIMARY KEY,
-    simulator_type_id TEXT,
-    simulator_type_name TEXT,
-    simulator_file_path TEXT,
-    project_file_path TEXT
-);
-CREATE TABLE result (
-    id TEXT PRIMARY KEY,
-    name TEXT,
-    project_name TEXT,
-    date TEXT
-);
-CREATE TABLE result_tag (
-    id TEXT PRIMARY KEY,
-    tag_id TEXT,
-    result_id TEXT,
-    FOREIGN KEY (result_id) REFERENCES result (id) ON DELETE CASCADE,
-    FOREIGN KEY (tag_id) REFERENCES tag (id) ON DELETE CASCADE
-);
-CREATE TABLE tag (
-    id TEXT PRIMARY KEY,
-    name TEXT
-);
-CREATE TABLE snapshot (
-    id TEXT PRIMARY KEY,
-    result_id TEXT,
-    date TEXT,
-    FOREIGN KEY (result_id) REFERENCES result (id) ON DELETE CASCADE
-);
-CREATE TABLE cross_section_snapshot (
-    id TEXT PRIMARY KEY,
-    snapshot_id TEXT,
-    cross_section_name TEXT,
-    b_display INT,
-    FOREIGN KEY (snapshot_id) REFERENCES snapshot (id) ON DELETE CASCADE
-);
-CREATE TABLE lane_snapshot (
-    id TEXT PRIMARY KEY,
-    cross_section_snapshot_id TEXT,
-    lane_number INT,
-    average_speed REAL,
-    traffic_volume INT,
-    a_display INT,
-    FOREIGN KEY (cross_section_snapshot_id) REFERENCES cross_section_snapshot (id) ON DELETE CASCADE
-);
-CREATE TABLE vehicle_snapshot (
-    lane_snapshot_id TEXT,
-    vehicle_type INT,
-    speed REAL,
-    FOREIGN KEY (lane_snapshot_id) REFERENCES lane_snapshot (id) ON DELETE CASCADE
-);""")
-        await self._connection.commit()
+        if not already_existed:
+            await self._connection.executescript("""PRAGMA foreign_keys = ON;
+    DROP TABLE IF EXISTS project;
```

ich wuerde dann mal die drop table statements weg machen. wenn wir das ausfuehren sollten ja nie tables existen und wenn welche doch existen ist dass ein error und wir wollen notified werden und nicht einfach silently droppen vor allem weil das ja data loss bedeutet